lib/omnibus-ctl.rb

```diff
@@ -57,6 +57,10 @@ def initialize(name, service_commands=true)
             :desc => "Show the configuration that would be generated by reconfigure.",
             :arity => 1
           },
+          "show-versions" => {
+            :desc => "Show the components and their versions comprising Chef Server.",
```

Comment:
This project is used by more than just chef server so we'd need to make this statement more generic.